bsmetadata/preprocessing_utils.py

```diff
@@ -80,3 +85,70 @@ def _extract_timestamp_from_url(self, url: str) -> Optional[str]:
         date = remove_improbable_date(date)
         date = str(date) if date is not None else None
         return date
+
+
+class EntityPreprocessor(MetadataPreprocessor):
+    """Metadata preprocessor for adding entity information."""
+
+    base_url = ".\preprocessing_scripts\entity_linking_files"
```

Comment:
Would it be possible to set the path in the preprocessor `__init__` arguments? So we don't have to download the files (of several gigas) to this exact location. I think this will make it easier to use the installed package smile